src/com/oltpbenchmark/benchmarks/tpcc/procedures/OrderStatus.java

```diff
@@ -71,10 +72,18 @@
       "   AND C_LAST = ? " +
       " ORDER BY C_FIRST");
 
-  private PreparedStatement ordStatGetNewestOrd = null;
-  private PreparedStatement ordStatGetOrderLines = null;
-  private PreparedStatement payGetCust = null;
-  private PreparedStatement customerByName = null;
+  private InstrumentedPreparedStatement ordStatGetNewestOrd = null;
+  private InstrumentedPreparedStatement ordStatGetOrderLines = null;
+  private InstrumentedPreparedStatement payGetCust = null;
+  private InstrumentedPreparedStatement customerByName = null;
+
+  public static void printLatencyStats() {
+    LOG.info("OrderStatus : ");
+    LOG.info("latency OrdStatGetNewestOrd " + ordStatGetNewestOrdSQL.getStats());
+    LOG.info("latency OrdStatGetOrderLines " + ordStatGetOrderLinesSQL.getStats());
+    LOG.info("latency PayGetCust " + payGetCustSQL.getStats());
+    LOG.info("latency CustomerByName " + customerByNameSQL.getStats());
+  }
```

Comment:
Should they be all in the same line?

Reply:
These are pretty long output lines. It would make them unreadable to have multiple of these on the same line.